addons/knobs/src/components/types/Select.tsx

```diff
@@ -3,17 +3,15 @@ import PropTypes from 'prop-types';
 
 import { Form } from '@storybook/components';
 
-type SelectTypeKnobValue = string;
+export type SelectTypeKnobValue = string | string[] | null;
```

Comment:
Are `select()` knobs always strings? Is `select(1, 2, 3, 4)` not possible? 

Also, if you allow nulll.. is the intention to pass in a null value as knob, like that `select(null, 'string')`? 
If yes, this could be useful too: `select(null, undefined, 'string')`

Reply:
Good points! You're right, technically numbers should be fine because they'd pass the strict eq. comparison. I had added `null` because it was in the example, and similarly `undefined` should work.

Updated.